core/src/elements/grenade.rs

```diff
@@ -12,11 +13,12 @@ pub fn install(session: &mut GameSession) {
 #[ulid = "01GPRSBWQ3X0QJC37BDDQXDN84"]
 pub struct IdleGrenade;
 
-#[derive(Clone, TypeUlid, Debug, Copy)]
+// #[derive(Clone, TypeUlid, Debug, Copy)]
+#[derive(Clone, TypeUlid, Debug)]
 #[ulid = "01GPY9N9CBR6EFJX0RS2H2K58J"]
 pub struct LitGrenade {
     /// How long the grenade has been lit.
-    pub age: f32,
+    pub damage_delay: Timer,
```

Comment:
Last change! Could you re-name this to `fuse_time`, and remove the comment, or update it with a new description.